.gitignore

```diff
@@ -0,0 +1,24 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
```

Comment:
Please change the PR message to reflect the changes that are present in the PR

Reply:
Done